editor/src/components/canvas/commands/set-elements-to-rerender-command.ts

```diff
@@ -78,3 +80,8 @@ export const runAppendElementsToRerender: CommandFunction<AppendElementsToRerend
     }]`,
   }
 }
+
+function uniqueElementPaths(paths: Array<ElementPath>) {
+  const pathSet = new Set(paths.map(EP.toString))
+  return Array.from(pathSet).map(EP.fromString)
```

Comment:
I _think_ you can use our util called `uniq` (which is a one liner `Array.from(new Set(array))`)

Reply:
ElementPaths should be referentially equal because when you call EP.fromString we return a cached reference

Reply:
Ok, I did not use uniq because I did not know about the referential equality